SofaKernel/modules/SofaBaseTopology/TriangleSetTopologyContainer.cpp

```diff
@@ -291,7 +291,6 @@ void TriangleSetTopologyContainer::createEdgesInTriangleArray()
                         << " [" << t[(j + 1) % 3] << ", " << t[(j + 2) % 3] << "]"
                         << " in triangle " << i;
                     m_edgesInTriangle.clear();
-                    return;
```

Comment:
Removing this return doesn't solve the real problem but override the security added to avoid using a non coherent topology. Indeed this is not perfect, before the return, the component should be set to Invalid.  I Will add that in a future PR. 
To really solve this problem check my comment in issue #1243 

I just discover there is no option to "disapprove" a PR 👎